minimesos/src/main/java/com/containersol/minimesos/mesos/ClusterArchitecture.java

```diff
@@ -276,7 +288,9 @@ public Builder withContainer(ClusterProcess container) {
          * Includes your own container in the cluster with a reference to another container of type T
          *
          * @param container container must extend from {@link ClusterProcess}. Functional, to allow you to inject a reference to another {@link ClusterProcess}.
-         * @param filter    A predicate that returns true if the {@link ClusterProcess} is of type T
+         * @param filter A predicate that returns true if the {@link ClusterProcess} is of type T
+         * @param <T> TODO(mwl)
+         * @return TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)